src/plugins/intel_cpu/src/nodes/pad.cpp

```diff
@@ -225,9 +222,9 @@ void Pad::PadExecutor::paramsInitialization(const PadAttrs& attrs,
     auto& srcMemPtr = srcMemory[DATA_ID];
     auto& dstMemPtr = dstMemory[DATA_ID];
     if (!dstMemPtr || !dstMemPtr->isDefined())
-        OPENVINO_THROW(errorPrefix, "has undefined source memory.");
+        OPENVINO_THROW("has undefined source memory.");
     if (!srcMemPtr || !srcMemPtr->isDefined())
-        OPENVINO_THROW(errorPrefix, "has undefined destination memory.");
+        OPENVINO_THROW("has undefined destination memory.");
```

Comment:
The same thing. Should we add at least the name of the executor class?

Reply:
Added